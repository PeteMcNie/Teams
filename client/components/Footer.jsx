import React from 'react'

const Footer = () => {
  const copyright = '2020 McNie Brothers'
  const author = 'P. R. McNie'

  return (
    <div className='footer'>
      <div>&copy; {copyright}</div>
      <div>{author}</div>
    </div>
  )
}

export default Footer
