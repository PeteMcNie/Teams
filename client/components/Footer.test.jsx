import React from 'react'

import { renderWithRedux } from '../testing/utils'

import Footer from './Footer'

test('Footer component matchs snapshot', () => {
  const { container } = renderWithRedux(<Footer />)
  expect(container).toMatchSnapshot()
})
