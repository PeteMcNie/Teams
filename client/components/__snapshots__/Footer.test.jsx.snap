// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matchs snapshot 1`] = `
<div>
  <div
    class="footer"
  >
    <div>
      Â© 
      2020 McNie Brothers
    </div>
    <div>
      P. R. McNie
    </div>
  </div>
</div>
`;
